patch.py

```diff
@@ -92,6 +92,24 @@ async def update_pd_org_price_plans():
 
     print(f'Updated {companies_updated} companies')
 
+@command
+async def update_pd_org_price_plans():
+    """
+    This patch sends a webhook to Pipedrive to update the price plan of all companies with a price plan
+    """
+    companies = await Company.filter(~Q(price_plan=None))
```

Comment:
why does this need the ~Q filter? is this the only way to do this in hermes

Reply:
I believe so, as you cant do `price_plan__is_null=False`

Reply:
![image](https://github.com/user-attachments/assets/93f6f3dc-aa17-48c1-a4ce-149f00923255)

Reply:
can do `await Company.exclude(price_plan=None)`, yields same result